MANIFEST.in

```diff
@@ -39,6 +40,14 @@ global-exclude .DS_Store
 global-exclude .git*
 global-exclude \#*
 
+global-exclude *.c
+global-exclude *.cpp
+global-exclude *.h
+
+global-exclude *.py[codx]
+global-exclude *.px[di]
+global-exclude *.pxi.in
```

Comment:
and remove this.

will submit a PR once get pip-test and conda-test to pass locally (built from sdist) with these changes